enterprise/app/executors/executors.tsx

```diff
@@ -288,47 +292,74 @@ export default class ExecutorsComponent extends React.Component<Props, State> {
-      if (this.state.nodes.length == 0) {
-        return (
-          <div className="empty-state">
-            <h1>No self-hosted executors connected.</h1>
-            <p>Self-hosted executors are enabled, but there are no executors connected.</p>
-            <p>Follow the instructions below to deploy your executors.</p>
-            <ExecutorSetup
-              user={this.props.user}
-              executorKeys={this.state.executorKeys}
-              executors={this.state.nodes}
-              schedulerUri={this.state.schedulerUri}
-            />
+    const defaultTabId = this.state.nodes.length > 0 ? "status" : "setup";
+    const activeTab = (this.props.path.substring("/executors/".length) || defaultTabId) as TabId;
+
+    return (
+      <>
+        <div className="tabs">
+          <div
+            className={`tab ${activeTab === "status" ? "selected" : ""}`}
+            onClick={this.onClickTab.bind(this, "status")}>
+            Status
           </div>
-        );
-      }
-      return (
-        <>
-          <h1>Self-hosted executors</h1>
-          <ExecutorsList executors={this.state.nodes} />
-          <hr />
-          <h1>Deploying additional executors</h1>
-          <ExecutorDeploy executorKeys={this.state.executorKeys} schedulerUri={this.state.schedulerUri} />
-        </>
-      );
-    } else {
-      return (
-        <div className="empty-state">
-          <h1>You're currently using BuildBuddy Cloud executors.</h1>
-          <p>See the instructions below if you'd like to use self-hosted executors.</p>
+          <div
+            className={`tab ${activeTab === "setup" ? "selected" : ""}`}
+            onClick={this.onClickTab.bind(this, "setup")}>
+            Setup
+          </div>
+        </div>
+        {activeTab === "status" && (
+          <>
+            {this.state.nodes.some((node) => !node.enabled) && (
+              <div className="callout warning-callout">
+                <AlertCircle className="icon orange" />
+                <div className="callout-content">
+                  <div>
+                    Some executors are disabled since your organization is using BuildBuddy Cloud executors. To fix
+                    this, enable self-hosted executors in settings.
+                  </div>
+                  <div>
+                    <FilledButton className="organization-settings-button">
+                      <a href="/settings/" onClick={linkHandler("/settings")}>
+                        Open settings
+                      </a>
+                    </FilledButton>
+                  </div>
+                </div>
+              </div>
+            )}
+            <ExecutorsList executors={this.state.nodes} />
+            {!this.state.nodes.length && this.props.user.selectedGroup.useGroupOwnedExecutors && (
+              <div className="empty-state">
+                <h1>No self-hosted Linux executors are connected.</h1>
```

Comment:
nit: should we remove the word `Linux` here?

Reply:
Good catch - done